### PR TITLE
test: Add some gherkin tests scenarios

### PR DESCRIPTION
**Pull request by romuleald**

### CODE REVIEW

File: test/features/end-of-game.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Feature: End of game
+
+  Scenario Outline: Game stop when a player reach <title> points
```
Comment (chipp972) [inline test/features/end-of-game.feature]: <score> plutôt ?

File: test/features/penalties.feature [hunk 2]
```diff
@@ -0,0 +1,41 @@
+Feature: Penalties
+
+  Scenario: Player take penalty and a new card is added to his set
+    Given 4 player
```
Comment (chipp972) [inline test/features/penalties.feature]: 4 players

Comment (yoannlg) [reply test/features/penalties.feature]: Flo sort de ce corp xD

Comment (romuleald) [reply test/features/penalties.feature]: done

**Pull request merged**